### ISSUE TITLE
Unit database doesn't show variations (website)

### ISSUE THREAD

**Issue by IskarJarak** [wesnoth/wesnoth#4753]
If I understood it correctly, variations were only used for visual diversity. With the Sand Scuttler variation to the Giant Scorpion we now have a variation in core that is relevantly different (massively different resistances) from the base unit. The unit database (https://units.wesnoth.org/1.15/mainline/en_US/mainline.html) does not display this variation at all though. Adding support for variations could be helpful.

**Comment by IskarJarak** [comment]
Just remembered that this would apply to walking corpses as well - I know they can all be seen as SotA-specific units, but seeing the various different movement costs and defenses by switching through the variations for the main unit could be another interesting use-case.

**Comment by cooljeanius** [comment]
> Just remembered that this would apply to walking corpses as well - I know they can all be seen as SotA-specific units, but seeing the various different movement costs and defenses by switching through the variations for the main unit could be another interesting use-case.

Yeah, I'm trying to set up a Walking Corpse spawner that randomizes its variations for one of my campaigns currently, and having a good viewer for all their variations on the website would be really helpful.
